config/locales/en.yml

```
@@ -159,6 +159,8 @@ en:
       already_registered: "This email address is already registered. Please log in to continue, or go back and use another email address."
     success:
       logged_in_succesfully: "Logged in successfully"
+    sessions:
+      signed_out: "Signed out successfully."
```

I see there's a similar key in Devise's translation file here and it's used if a logout request is received when the user is already logged out. We may as well add it here.

```suggestion
      signed_out: "Signed out successfully."
      already_signed_out: "Signed out successfully."      
```